PocketKit/Sources/Sync/ApolloClientProtocol+Extensions.swift

```diff
@@ -72,17 +68,28 @@ public extension ApolloClientProtocol {
         }
     }
 
-    private static func handleServerError(_ error: Error) -> Bool {
+    /// Takes a GraphQl response error and logs it to Sentry. If the error code belongs
+    /// to a specified set of errors, also posts a notification containing the status code
+    private static func handleServerError(_ error: Error,
+                                          filename: String,
+                                          line: Int,
+                                          column: Int,
+                                          funcName: String) {
 
         // Codes we wish to notify the user about
-        let serverErrorCodes = [429, 500, 503]
+        let notifiableErrorCodes = [429, 500, 503]
 
         guard let responseError = error as? ResponseCodeInterceptor.ResponseCodeError,
               case .invalidResponseCode(let response, _) = responseError,
-              let code = response?.statusCode,
-              serverErrorCodes.contains(code) else { return false }
+              let code = response?.statusCode else {
+            Log.capture(message: "GraphQl Error - unknown response/status code.", filename: filename, line: line, column: column, funcName: funcName)
```

Comment:
should we capture general info we get from `error` here too while we have it? 

GraphQl Error - unknown response/status code \(error).

Reply:
Good point @cyndichin ! Actually the existing interceptor carries a bunch of info that we might want to carry over. thanks for spotting this!